### DISCUSSION TITLE
Is it possible to create alert?

### DISCUSSION THREAD

**Discussion by mahmutefe** [grafana/grafana-infinity-datasource#283] [Q&A] [answered]
I tried to create an alert via infinity plugin and  received the below error:

Test API: https://thingspeak.com/channels/38629/feed.json

![image](https://user-images.githubusercontent.com/5229401/158916990-61108764-28a4-4912-8eb1-e98f225ee39c.png)

**Comment by yesoreyeram** [comment, marked as answer]
1.0.0-beta.1 is now **[available](https://github.com/yesoreyeram/grafana-infinity-datasource/releases/tag/v1.0.0-beta.1)** with alerting support. 

With JSON query type, one need to choose `backend` as a parser to get the alerts working. Example shown below for setting up alert.

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/153843/188924894-4b1f3d3d-2857-484e-916c-d09cb8d4dd5c.png">

Example API used : https://thingspeak.com/channels/38629/feed.json

**Comment by mahmutefe** [comment]
Thanks for your efforts. When will you officially release it? 
Definitely. It's a great job. Cong.
![image](https://user-images.githubusercontent.com/5229401/194794507-cb89d679-496c-4c50-89de-0c6457e6f334.png)

**Reply by yesoreyeram** [reply]
It is already released for JSON types. Csv support for alerts is coming in 1.1

**Reply by yesoreyeram** [reply]
Thank you for the support and kind words.
